action.yml

```diff
@@ -1,7 +1,7 @@
 name: nightfalldlp
 description: check PR's for sensitive information
 runs:
-  using: 'Docker'
+  using: 'docker'
```

Comment:
does casing matter? or was the original `Docker` causing an issue?

Reply:
it wasn't causing an issue but my linter picked up that it was an invalid value. The lowercase `docker` is the appropriate value (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing-2)